prereise/gather/winddata/impute.py

```diff
@@ -137,3 +137,17 @@ def gaussian(data, wind_farm, inplace=True, curve="state"):
 
     if not inplace:
         return data_impute
+
+
+def linear(data, inplace=True):
+    """Given a 2D array, linearly interpolate any missing values column-wise.
+
+    :param numpy.array/pandas.DataFrame data: data to interpolate.
+    :param bool inplace: whether to modify the data inplace or return a modified copy.
+    :return: (*None/pandas.DataFrame*) -- if ``inplace`` is False, data frame with
+        missing entries imputed.
+    """
+    data_impute = data if inplace else data.copy()
+    data_impute[:] = pd.DataFrame(data_impute).interpolate()
+    if not inplace:
+        return data_impute
```

Comment:
Not sure whether we can simply `return pd.DataFrame(data).interpolate(inplace=inplace)` here?

Reply:
I don't think we can, since `pd.DataFrame(data)` creates a new object. That's why I have the `data_impute[:] = ...` line: so we can modify the data contained in this object without creating a new one.

Reply:
```python
import numpy as np
import pandas as pd
>>> a = np.array([[1, float("nan"), 4, 5], [0, float("nan"), float("nan"), 4]]).T
>>> a
array([[ 1.,  0.],
       [nan, nan],
       [ 4., nan],
       [ 5.,  4.]])
>>> pd.DataFrame(a).interpolate(inplace=True)
>>> a
array([[ 1.,  0.],
       [nan, nan],
       [ 4., nan],
       [ 5.,  4.]])
```